### DISCUSSION TITLE
Azithromycin PBPK Model project file

### DISCUSSION THREAD

**Discussion by harishkaushikbugworks** [Open-Systems-Pharmacology/Forum#807] [Questions & Problems]
Hello All,
Can any one help me with sharing the Azithromycin PK SIM project file, iam referring to 
https://pdfs.semanticscholar.org/c58c/83cfabd61a94bc4ed53e7e1a8b6a35557d6f.pdf
link above
Author has built PBPK model using PKSIM , when iam trying to do the same iam unable to get the fiting
i am using same literature data as mentioned in the thesis, despite that iam not getting the same
[New folder.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/3818248/New.folder.zip)

attaching project and thesis pdf copy

Regards
Harish

**Comment by harishkaushikbugworks** [comment]
Hello All,
I could fit the model of azithromycin now, but iam trying to fit my test & reference oral suspension data using dissolution data, i could reasonably predict reference formulation oral data from weibull dissolution data but there is under prediction for test formulation data

I need your help in optimising this model further
iam attching Test & Ref sim fitting obtained and diso data of test and reference as well invivo data

Thanks
Harish
![ref sim fitting](https://user-images.githubusercontent.com/49284633/68472842-6cd42f00-0247-11ea-9726-14e9de445ec3.jpg)
[test and ref disso.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/3824293/test.and.ref.disso.xlsx)
![test sim fitting](https://user-images.githubusercontent.com/49284633/68472843-6d6cc580-0247-11ea-9090-544e25a04264.jpg)
[TEST ND REF.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/3824303/TEST.ND.REF.xlsx)

**Comment by LuzNunez** [comment]
Hi, 
I am with the same problem!
Could you solve it? 

Thanks
Luz

**Comment by StephanSchaller** [comment]
@harishkaushikbugwork, sorry this thread was not answered. Did you solve your issue? If so, it would be great if you could share your solution here in GitHub.

Another publication on this compound can be found here: https://link.springer.com/article/10.1007%2Fs40268-016-0170-8

Thanks,
Stephan
